data/hai/unfettered weapons.txt

```diff
@@ -32,7 +32,7 @@ outfit "Charge Gas Grenadier"
 		"firing energy" 100
 		"firing heat" 80
 		"firing force" 20
-	description `The Unfettered have had no qualms with tweaking Hai technology for their own uses, and this weapon is a good example of that. Essentially an ionic grenade launcher, the simple fabricated canisters fired are under high pressure and likely to explode long before they reach their target, but the unfettered's actions to reduce the size as much as possible by cutting out some of the tanks have resulted in a potent fighter weapon.`
+	description `The Unfettered have made a point of tweaking and improving Hai technology for their own uses, and this weapon is a good example of this strategy. Essentially an ionic grenade launcher, this weapon fires Cargo Gas canisters fired are under high pressure, making them likely to explode long before they reach their target. By re-engineering the Charge Gas Sprayer into a smaller weapon and cutting out an integrated storage tank, the Unfettered have created a potent fighter weapon out of a system typically reserved for larger ships.`
```

Comment:
"Cargo Gas"? I think you mean "Charge Gas".

Reply:
```suggestion
	description `The Unfettered have made a point of tweaking and improving Hai technology for their own uses, and this weapon is a good example of this strategy. Essentially an ionic grenade launcher, this weapon fires Carge Gas canisters fired are under high pressure, making them likely to explode long before they reach their target. By re-engineering the Charge Gas Sprayer into a smaller weapon and cutting out an integrated storage tank, the Unfettered have created a potent fighter weapon out of a system typically reserved for larger ships.`
```

Reply:
Well, you're getting closer. It says "Carge Gas" now.

Reply:
AAAAAARGH